packages/oui-typography/typography.less

```diff
@@ -14,6 +15,7 @@ abbr[data-oui-tooltip] {
   cursor: help;
 }
 
+// Headings
 h,
```

Comment:
I'm not sure that `h` is a valid selector. Am I wrong?

Reply:
Look what's doing the mixin behind :)

Reply:
`&@{index}` alright! Thanks to point out.